Oneirocom/Magick
# Small fixes 1

Merge after #1311 

## What Changed:

- Added local featyre flag for ENV to avoid using posthog locally.  Posthog actually overrides console logs and doest not give stack traces or proper line number references.
- Always log node errors to the IDE console even when the node is not told to log.

## How to test:

- Check if drawer doesnt show up when no env variable is set
-  add proper feature flag ENV variable to client env to ensure it can be turned on.

## Additional information:

Any other information that might be useful while reviewing.
